src/perf-tests/perftest-stack-airline/lambda/index.js

```diff
@@ -0,0 +1,38 @@
+'use strict'
+
```

Comment:
Could you add some comments on what this function does? It will help those looking to understand why it's necessary